go/client/cmd_simplefs_copy.go

```diff
@@ -67,8 +72,12 @@ func (c *CmdSimpleFSCopy) Run() error {
 		dest, err = makeDestPath(c.G(), ctx, cli, src, c.dest, isDestDir, destPathString)
 		c.G().Log.Debug("SimpleFSCopy %s -> %s, %v", pathToString(src), pathToString(dest), isDestDir)
 
-		if err == TargetFileExistsError && c.interactive == true {
-			err = doOverwritePrompt(c.G(), pathToString(dest))
+		if err == TargetFileExistsError {
+			if c.interactive == true {
+				err = doOverwritePrompt(c.G(), pathToString(dest))
+			} else if c.force == true {
+				err = nil
+			}
```

Comment:
Since `--interactive` and `--force` are kind of opposites, does it make sense to return a parse error if they're both true?